kahuna/public/js/edits/image-editor.js

```diff
@@ -29,16 +29,26 @@ imageEditor.controller('ImageEditorCtrl', [
     'editsApi',
     'imageService',
     'collections',
+    'mediaApi',
 
     function($rootScope,
              $scope,
              $timeout,
              editsService,
              editsApi,
              imageService,
-             collections) {
+             collections,
+             mediaApi) {
 
     var ctrl = this;
+    ctrl.canUndelete = false;
+    ctrl.showUndeleteMsg = false;
```

Comment:
can this be removed, I can't see it being used (nor set to anything other than false)